src/xenia/kernel/kernel_state.h

```diff
@@ -236,6 +236,13 @@ class KernelState {
   friend class XObject;
 };
 
+enum : XNotificationID {
+  kSystemUI = 0x00000009,
```

Comment:
For `enum` rather than `enum class` (that have their values leaked into the scope of where they're defined), the values themselves need to be prefixed like `kXNotificationIDSystemUI`, to state clearly what they are in their usage locations, and to avoid potential collisions with unrelated values.

Reply:
Okay. I have changed the `enum` values accordingly.